tcf_website/views/auth.py

```diff
@@ -6,19 +6,22 @@
 from django.contrib.auth import logout as auth_logout
 from django.contrib import messages
 from django import forms
-
+from .browse import browse
 
 def login(request):
     """Login view."""
     if request.user.is_authenticated:
-        return redirect('browse')
-    return render(request, 'login/login.html')
-
+        messages.success(request, "Logged in successfully!")
+        return redirect('profile')
+    return browse(request)
```

Comment:
Just curious — when does this return get reached?

Reply:
When the user isn't authenticated and they try to go to `/login`